meshpy/cosserat_curve/cosserat_curve.py

```diff
@@ -440,7 +440,7 @@ def get_pyvista_polyline(self) -> pv.PolyData:
         cell = np.insert(cell, 0, self.n_points)
         poly_line.lines = cell
 
-        base = [[], [], []]
+        base: list[list[np.ndarray]] = [[], [], []]
```

Comment:
This whole construct was a little bit shady, I replaced this with a `numpy` array.